### PR TITLE
Qr code checking

### PR DESCRIPTION
**Pull request by nopjmp**
This will check for https://discordapp.com/ra QR codes and remove the messages. It will log to the configured messages channel.

### CODE REVIEW

File: src/events/message/message.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import Config from 'config'
+
+export default class messageEvent {
+  constructor() {
+    this.name = 'message'
+  }
+
+  async handle(message) {
+    // ignore DMs
+    if (!message.channel.guild) return
```
Comment (flutterflies) [inline src/events/message/message.js]: Minor Style nit, but can you keep the braces on one line `if`s like this
```js
if (!message.channel.guild) { return }
```

I'll try to setup linting to enforce this in the near future

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import CommandHandler from 'commands/command-handler'
+import CommandHandler from './commands/command-handler'
```
Comment (flutterflies) [inline src/index.js]: what's the reason for this change? Was the module resolver stuff not working correctly?

Comment (nopjmp) [reply src/index.js]: Nodejs13 applies the experimental ESM resolver now and it broke. This change was a mistake and not to be added to the repo, I'll revert it when I update the other style things you mentioned.
